lib/elixir/src/elixir_erl_pass.erl

```diff
@@ -599,6 +632,36 @@ translate_remote(Left, Right, Meta, Args, S) ->
       {{call, Ann, {remote, Ann, TLeft, TRight}, TArgs}, SC}
   end.
 
+merge_pairs(List1, List2) ->
```

Comment:
Is this to avoid warnings? If so, one option is to move the warning to Elixir-land and ignore the warnings that come from the Erlang side of things.

Reply:
I moved them, but I'm not sure if I should add tests covering the fact that the maps can have repeated keys